core/src/main/java/org/apache/accumulo/core/client/PluginEnvironment.java

```diff
@@ -33,10 +35,7 @@
  */
 public interface PluginEnvironment {
 
-  /**
-   * @since 2.1.0
-   */
```

Comment:
Should preserve the since tag, since it's an inner-interface, and has its own apidoc page as a separate type.

Reply:
Thanks, updated for the next commit.